docs/openapi-backend.yaml

```diff
@@ -0,0 +1,393 @@
+openapi: 3.0.3
+info:
+  title: IO Sign - Backend API
+  version: 0.0.1
+servers:
+  - url: https://app-backend.io.italia.it/api/v1/fci
+    description: Production
+security:
+  - Bearer: []
+paths:
+  /signature-requests/{id}:
+    get:
+      operationId: getSignatureRequestById
+      summary: Get a Signature Request By Id
+      tags:
+        - Signature Request
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            description: Entity Id
+            format: ulid
+            example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
+          required: true
+      responses:
+        "200":
+          description: The Signature Request detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureRequestDetailView"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+
+  /qtsp/clauses:
+    get:
+      operationId: getQtspClausesMetadata
+      summary: Get clauses list and MRC from QTSP
+      tags:
+        - QTSP Clauses
+      responses:
+        "200":
+          description: The QTSP clauses list
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QtspClausesMetadata"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+
+  /qtsp/clauses/filled_document:
```

Comment:
```suggestion
  /qtsp/clauses/filled-document:
```

Reply:
@gunzip if `filled_document` is a property it should be written in `snake_case`, if it is a collection it is ok to mantain the `kebab-case`